src/lib/addons/slack.js

```diff
@@ -74,6 +84,15 @@ class SlackAddon extends Addon {
         return tags.filter(tag => tag.type === 'slack').map(t => t.value);
     }
 
+    generateStaleText({ createdBy, data }, isStale) {
```

Comment:
I am really happy to take some feedback on these texts. Was basically just writing something, without thinking much.